sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -154,8 +156,15 @@ object AlluxioUtils extends Logging {
       alluxioCmd = conf.getAlluxioCmd
       checkAlluxioNotSupported(conf)
 
-      if (!isInit) {
-        if (conf.getAlluxioAutoMountEnabled) {
+      if (isConfiguredReplacementMap(conf)) {
+        // replace-map is enabled, if set this will invalid the auto-mount
+        if (!isInitReplaceMap) {
+          alluxioPathsToReplaceMap = getReplacementMapOption(conf)
```

Comment:
so this also means that the replacement map can't be change during runtime, which is fine but might want to document.  It would be weird to change during runtime anyway since user has to mount them